cinema/tests/test_cinema_hall_api.py

```diff
@@ -116,9 +116,13 @@ def test_patch_cinema_hall(self):
         self.assertEqual(CinemaHall.objects.get(id=1).name, "Green")
 
     def test_delete_cinema_hall(self):
+        print("Before DELETE request:", CinemaHall.objects.all())
```

Comment:
Unnecessary print statement. Consider removing it to keep the test output clean.